app/assets/stylesheets/provider/_tables.scss

```diff
@@ -102,6 +102,9 @@ table.data {
   }
 
   tbody {
+    td:first-child {
+      word-break: break-all;
```

Comment:
This is a pretty generic style declaration. Might be good but why only on the first child?

Reply:
Because the first child: **Metric names**, is the only one with long names.
Also, it is nested inside table.data, so it's not so generic

Reply:
True, doing some changes...